styles/youtube/catppuccin.user.css

```diff
@@ -424,925 +441,353 @@
       --yt-button-payment-background-color: @accent-color !important;
     }
 
-    yt-icon.ytd-pinned-comment-badge-renderer * {
-      fill: @text !important;
-    }
-
-    .yt-spec-button-shape-next--size-m.yt-spec-button-shape-next--segmented-start::after {
-      background: @overlay0 !important;
-    }
-
-    .html5-ypc-purchase {
-      background: @accent-color !important;
-      color: @mantle !important;
-      &:hover {
-        background: lighten(@accent-color, 10%) !important;
-      }
-    }
-
-    .html5-ypc-description,
-    .html5-ypc-title,
-    .html5-ypc-action-heading {
-      color: @text;
+    ytd-author-comment-badge-renderer {
+      --yt-basic-background-color: @surface0 !important;
+      --yt-basic-foreground-title-color: @surface0 !important;
+      --ytd-author-comment-badge-background-color: @surface0 !important;
+      --ytd-author-comment-badge-name-color: @text !important;
+      --ytd-author-comment-badge-icon-color: @text !important;
     }
 
-    yt-notification-action-renderer[darker-dark-theme]
-      tp-yt-paper-toast.yt-notification-action-renderer {
-      background: @overlay0 !important;
+    /* Ambient mode */
+    #cinematics {
+      mix-blend-mode: lighten;
     }
 
-    /* volume slider */
-    .ytp-volume-slider-handle {
-      background: @accent-color !important;
-      &::before {
-        background: @accent-color !important;
-      }
+    .ytp-settings-button.ytp-hd-quality-badge::after,
+    .ytp-settings-button.ytp-hdr-quality-badge::after,
+    .ytp-settings-button.ytp-4k-quality-badge::after,
+    .ytp-settings-button.ytp-5k-quality-badge::after,
+    .ytp-settings-button.ytp-8k-quality-badge::after,
+    .ytp-settings-button.ytp-3d-badge-grey::after,
+    .ytp-settings-button.ytp-3d-badge::after {
+      background-color: @accent-color;
     }
 
-    /* new videos dot */
-    #newness-dot {
-      background-color: @accent-color !important;
-    }
+    /* Icons */
 
-    /* subtitle */
-    .ytp-chrome-controls .ytp-button[aria-pressed]::after {
-      background-color: @accent-color !important;
+    [fill="red"],
+    [fill="#F00"],
+    [fill="#FF0000"] {
+      fill: @accent-color !important;
     }
 
-    /* menu items */
-    /* checkbox subitem */
-    .ytp-menuitem-toggle-checkbox {
-      background-color: @overlay1;
-
-      &::after {
-        background-color: @subtext0;
-      }
+    [fill="white"] {
+      fill: if(@lookup = latte, @base, @text) !important;
```

Comment:
This seems to break the live indicator, since its bg is `@accent-color` it becomes unreadable on dark themes.

Reply:
I think I've fixed it if I understood you correctly. Can you check now?